setup.cfg

```diff
@@ -1,5 +1,76 @@
 [metadata]
-description-file = README.md
+name = textX
+author = Igor R. Dejanovic
+author_email = igor.dejanovic@gmail.com
+license = MIT
+description = Meta-language for DSL implementation inspired by Xtext
+keywords = parser, meta-language, meta-model, language, DSL
+url = https://github.com/textX/textX
+long_description = file: README
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: Science/Research
+    Topic :: Software Development :: Interpreters
+    Topic :: Software Development :: Compilers
+    Topic :: Software Development :: Libraries :: Python Modules
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+packages = textx, textx.scoping, textx.cli
+zip_safe = False
+install_requires =
+    Arpeggio>=1.9.0
+    click==7.0
+include_package_data = True
+package_dir =
+    textx = textx
+setup_requires =
+    wheel
+
+[options.extras_require]
+dev =
+    textX-dev
```

Comment:
This is probably ok, but I had these errors above: Is it ok that "textX" (dev) depends on "textX-dev" here and "textX-dev" in turn depends on "textX" (a circular dependency?).

Reply:
Yup, good question. It should be ok. textX has textX-dev as a development dependency, so you need to pull in `dev` dependency if you want to use additional textX dev tooling (currently just `startproject` command but I expect it to grow over time). One way would be to install `textX-dev` and pull `textX` as a dependency but I thought that it might be nicer to install textX directly and if additional dev tooling is needed than you could use `textX[dev]` in the same way you could use `textX[test]` to pull in test dependencies. It seems more consistent to me, and the helper scripts seems to be more consistent across the  projects.